nixos/modules/services/backup/sanoid.nix

```diff
@@ -195,6 +195,13 @@ in
       after = [ "zfs.target" ];
       startAt = cfg.interval;
     };
+
+  # Put those packages in the global environment
+  # so that syncoid can find them when targeting this host through ssh.
+  environment.systemPackages = [
+    pkgs.lzop
+    pkgs.mbuffer
+  ];
```

Comment:
```suggestion
    # Put those packages in the global environment
    # so that syncoid can find them when targeting this host through ssh.
    environment.systemPackages = with pkgs; [
      lzop
      mbuffer
    ];
```

Reply:
Thanks!